src/app/model/EditableImageDataProperty.ts

```diff
@@ -0,0 +1,35 @@
+import { FormControl } from '@angular/forms';
+
+export type InputType =
```

Comment:
Do we use all these types?

Reply:
For now no, but I left there all values possible to specify fot HTML input type.